gov_uk_dashboards/components/plotly/side_navbar.py

```diff
@@ -5,7 +5,7 @@
 
 
 def side_navbar(
-    links, identifier: Optional[str] = None, nav_id: Optional[str] = "navbar-section"
+    links, identifier: Optional[str] = None, nav_id: str = "navbar-section"
```

Comment:
Isn't any parameter that has a default optional? Or am I thinking about it wrong...?

Reply:
It is, adding the word optional allows you to pass in a none and we don't want to pass in a none for 
our ID